assets/src/application/ui/display/EntityListLegend/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Row, Col } from 'antd';
+
+import { EspressoIcon, Icon } from '@appDisplay/espressoIcon';
+import { LegendConfig } from './types';
+import './style.scss';
+
+interface Props {
+	legendConfig: LegendConfig;
+}
+
+const EntityListLegend: React.FC<Props> = ({ legendConfig }) => {
+	const { icons, swatches } = legendConfig;
+	return (
+		<Row className='ee-row'>
+			<Col span={12}>
+				<dl className='ee-list-table-legend'>
+					{Object.entries(icons).map(([icon, text]) => {
+						return (
+							<dt>
+								<EspressoIcon icon={icon} svgSize={18} />
```

Comment:
@manzoorwanijk I do get this error here, any hints or TS voodoo fixes?

![image](https://user-images.githubusercontent.com/1477453/76876167-3738f580-687a-11ea-925c-6fb9bd98a0df.png)